src/main/controllers/UserSsoController.ts

```diff
@@ -0,0 +1,54 @@
+import { AuthedRequest } from '../interfaces/AuthedRequest';
+import { Response } from 'express';
+import { RootController } from './RootController';
+import asyncError from '../modules/error-handler/asyncErrorDecorator';
+import autobind from 'autobind-decorator';
+import { USER_DETAILS_URL } from '../utils/urls';
+import { PageError } from '../interfaces/PageData';
+import { isEmpty } from '../utils/utils';
+import { MISSING_OPTION_ERROR, USER_DISABLE_SSO_ERROR } from '../utils/error';
+import { User } from '../interfaces/User';
+
+
+@autobind
+export class UserSsoController extends RootController{
+
+  @asyncError
+  public post(req: AuthedRequest, res: Response) {
+    return req.scope.cradle.api.getUserById(req.body._userId)
+      .then(user => {
+        switch(req.body.confirmSso) {
+          case 'true':
+            return this.removesso(req, res, user);
```

Comment:
Non standard casing for method name. It should be removeSso() and not removesso()

Reply:
fixed